src/main/java/gregicadditions/recipes/GARecipeAddition.java

```diff
@@ -5427,6 +4349,16 @@ public static void init2() {
                 .EUt(30)
                 .duration(1160)
                 .buildAndRegister();
+
+        // Chain Fixing
+        // Gold Dust Centrifuging
+        CENTRIFUGE_RECIPES.recipeBuilder().duration(160).EUt(20)
+                .input(dust, Netherrack)
+                .chancedOutput(OreDictUnifier.get(dustTiny, Materials.Redstone, 1), 5625, 850)
+                .chancedOutput(OreDictUnifier.get(dustTiny, PreciousMetal), 625, 500)
```

Comment:
```suggestion
                .chancedOutput(OreDictUnifier.get(dustTiny, PreciousMetal, 2), 625, 500)
```
I think this could use a small buff because with Gold Dust, it is useful immediately, but this requires more processing to get it to gold.